unfoldingWord/translationCore
# Remove Highlighting From Verse Check When Not Applicable

#### This pull request addresses:
This PR makes the select mode area the only place where a user can select/highlight when using the checking tool.


#### How to test this pull request:
`git checkout enhancement/jay/remove-highlight/2295` in Verse Check
Try highlighting the areas specified in #2295 and make sure they are not selectable/highlightable


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2306)
